### ISSUE TITLE
There is a failed activity log when pasting one regular blob container

### ISSUE THREAD

**Issue by v-kellyluo** [Microsoft/AzureStorageExplorer#5905]
**Storage Explorer Version:** 1.25.0-dev
**Build Number:** 20220706.1
**Branch:** main
**Platform/OS:** Windows 10/Linux Ubuntu 20.04/MacOS Monterey 12.4 (Apple M1 Pro)
**Architecture:** ia32\x64
**How Found:** From running test case
**Regression From:** Previous release (1.24.3)
 
## Steps to Reproduce ##

1. Expand one storage account -> Blob Containers.
2. Create a blob container -> Right click the blob container -> Click 'Copy Blob Container'.
3. Paste the blob container to another storage account.
4. Check whether there is a successful activity log.

## Expected Experience ##
There is a successful activity log.

## Actual Experience ##
There is a failed activity log.
![image](https://user-images.githubusercontent.com/87792676/177499624-047984e4-a1ba-4ba5-8981-db25ddcf2eb1.png)

## Additional Context ##

1. This issue doesn't reproduce if assigning 'Storage Blob Data Contributor' role to source blob container.
2. This issue doesn't reproduce if the source blob container is under one SAS attached storage account.

**Comment by JasonYeMSFT** [comment]
Could you please check the following things from 1.24.3 and the current build and make a comparison list?

- Does the source container has key credential?
- Does the source container has auth credential?
- Does the source container has SAS credential?
- If you look at the text on clipboard after copying the source container, what credentials are there in the connection object on the clipboard?

Then let's try to see if the same type of source can have copy working.

If the source container only has auth credential and its rbac permissions doesn't include reading blobs then this error is expected. No matter if we use a user delegation SAS or direct Azure AD credential, it will eventually go check the user's rbac roles and will fail.

**Comment by v-kellyluo** [comment]
@JasonYeMSFT , 

1. Verified this issue on the latest build main/20220707.4, it doesn't reproduce, and no matter pasting ADLS Gen2 blob containers or regular blob containers, there is no need to assign rbac role to source blob containers. 
2. And I have no idea to check the following issues, could you please give some guides?

![image](https://user-images.githubusercontent.com/87792676/177677262-14624973-4e09-4650-8077-08ee39dd007f.png)

**Comment by v-kellyluo** [comment]
@JasonYeMSFT ,
This issue doesn't reproduce on the build main/20220708.4. I compared the same source blob container between these two builds, on the build main/20220708.4, the source blob container has 'sas' and 'auth' credentials, on the build main/20220706.1, the source blob container only has 'auth' credential, so this error should be expected.
Not sure what changed, on the reproduced build, I unchecked the setting 'Disable usage of key' and enabled 'Allow storage account key access' on portal, the source blob container still only has the 'auth' credential.
